api/src/main/java/org/openmrs/module/reporting/cohort/definition/evaluator/SqlCohortDefinitionEvaluator.java

```diff
@@ -57,6 +63,14 @@ public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationCon
 			return ret;
 		}
 
+		final Boolean includeChildLocations = (Boolean) context.getParameterValue("includeChildLocations");

```

Comment:
I don't think there is any need to modify the parameter values in the context here.  The SqlQueryBuilder simply takes in a Map<String, Object>.  It would better to clone the parameter values off of the context to start, then modify this Map of parameter values to have the list of locations if necessary, and then pass this into the query builder.  If you modify the context, that might have the affect of invalidating the cache, and other side effects, so it is best to avoid that if it is not needed.